Add the November and December 2021 SC update (#111)
python/steering-council: updates/2021-12-steering-council-update.md

# December 6

- The Steering Council met with Łukasz to discuss his progress and also
  introduced Ee to the meeting since Ee will be taking PSF lead on supporting

>   Łukasz going forward. Łukasz updated the team on CLA work, testing Irit's PR,

Reviewer: Irit's PR for what? (Irit has authored many PRs. :-)

>   adding a new triagger, and giving a keynote at the PyCode Conference.

Reviewer: ```suggestion
  adding a new triager, and giving a keynote at the PyCode Conference.
```

>   adding a new triagger, and giving a keynote at the PyCode Conference.

Reviewer 2: ```suggestion
  adding a new triager, and giving a keynote at the PyCode Conference.
```

- The Steering Council discussed the issue Jason posted about deprecating

>   importlib.resources legacy API. It was decided that Brett would respond to

Reviewer: Was this an issue in the public SC repo? Maybe link to it? (In general maybe link more often to discussions? A year from now people might not have enough context to understand what the minutes are saying.)

  the comment stating that Jason is following policy and that the SC is
  satisfied, but that the larger discussion on changing the policy should
  happen on python-dev. The group also discussed how Python's deprecation
  policy impacts users and library authors.

> - Group said their byes on Ewa's last day.

Reviewer: ```suggestion
- The group said their goodbyes on Ewa's last day.
```

# December 6

- The Steering Council met with Łukasz to discuss his progress and also
  introduced Ee to the meeting since Ee will be taking PSF lead on supporting
  Łukasz going forward. Łukasz updated the team on CLA work, testing Irit's PR

>   for Exception Grops, adding a new triager, and giving a keynote at the PyCode

Reviewer 3: ```suggestion
  for Exception Groups, adding a new triager, and giving a keynote at the PyCode
```

  posted](https://github.com/python/steering-council/issues/89) about
  deprecating importlib.resources legacy API. It was decided that Brett would
  respond to the comment stating that Jason is following policy and that the SC
  is satisfied, but that the larger discussion on changing the policy should
  happen on python-dev. The group also discussed how Python's deprecation
  policy impacts users and library authors.
- The group said their goodbyes on Ewa's last day.

# December 13

- The Steering Council met with Ezio. There is no full import from BPO yet, but
  he will create one this week. The final migration is still tentatively
  scheduled for mid-January. Ezio will loop in Thomas on meetings with GitHub,
  which Ewa previously attended, or other SC members if Thomas can't make it.
- The SC prepared for a Talk Python podcast recording after the meeting.

# December 13 (handoff meeting)

- The old Steering Council and the New Steering Council met for the handoff
  meeting.
- The SC discussed several aspects on how it has operated so far.
- The SC received an updated from Thomas on the status on the search for a new
  PSF director.

> - The SC brielfly reflected and discussed about the diversity in the Steering

Author: ```suggestion
- The SC briefly reflected and discussed the diversity in the Steering
```
